### ISSUE TITLE
Nice Job

### ISSUE THREAD

**Issue by menghuanlater** [luyug/GradCache#1]
很棒的工作，正好解决了我所遇到的难题

**Comment by here101** [comment]
> 很棒的工作，正好解决了我所遇到的难题

请问你方便放出你的关于这个训练过程的部分代码吗，我英语阅读能力有限，没看太懂这个readme中的如何使用。
我不知道如何与一般训练过程中的loss计算,loss.backward,optimizer.step().
谢谢！
